tests/test_back_filters.js

```diff
@@ -25,7 +26,7 @@ function createServer(t) {
     t.pass('server received a request')
 
     response.writeHead(200)
-    response.write(`server served a response at ${new Date()}`)
+    response.write(`server served a response at ${new Date().getTime()}`)
```

Comment:
The `nockBack passes filteringPath options` test makes two requests and asserts that the second response comes from the recorded response of the first request, however, the response body was only showing resolution of one second so it would be easy to get a false positive from the assertion. 
Honestly, it should probably be returning a proper random value, but using milliseconds should be plenty good enough for this use case. 

example before
`'server served a response at Sun Jun 23 2019 07:13:56 GMT-0600 (Mountain Daylight Time)'`
after
`'server served a response at 1561295815555'`

Reply:
We can use `Date.now()` instead of `new Date().getTime()`.

For a more random string I like to use `Math.random().toString(36).substr(2)`